### PR TITLE
Mandatory fields sign

### PR DESCRIPTION
**Pull request by nnn**
Closes https://github.com/apinf/platform/issues/1171

### CODE REVIEW

**Comment by marla-singer** [comment]
@nnn Not bad!
But Sing up form doesn't have the star
joxi_screenshot_1480519890082

**Comment by nnn** [comment]
@marla-singer bypass:
```
.at-pwd-form label:after {
  content: '*';
  font-size: 0.8em;
  color: @brand-danger;
}
```

File: core/client/style/global.less [hunk 1]
```diff
@@ -15,6 +15,19 @@ a {
   text-decoration: none;
 }
 
+[data-required] label:after {
```
Comment (marla-singer) [inline core/client/style/global.less]: You can unite both rules and save it as 
```css
[data-requerid] label:after, 
.at-pwd-form label:after {
...
}
```

**Comment by marla-singer** [comment]
@nnn Merge it

**Pull request merged**